charts/filsnap/templates/snapshots/secret.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.snapshots.uploads.enabled }}
```

Comment:
Can you make secret creation optional here?
Helm values can be read for anyone able to invoke helm commands on a release, which may be fine sometimes, but not always